plugins/VisitFrequency/API.php

```diff
@@ -120,7 +120,7 @@ protected function unprefixColumns(array $requestedColumns, $suffix)
     protected function prefixColumns($table, $period, $suffix)
     {
         $rename = array();
-        foreach (APIVisitsSummary::getInstance()->getColumns($period) as $oldColumn) {
+        foreach ($table->getColumns() as $oldColumn) {
```

Comment:
simply renaming all columns should have the same effect, but makes the `getColumns` method obsolete